www/mikt04/sp2/assets/js/main.js

```diff
@@ -0,0 +1,341 @@
+$(document).ready(function () {
+
+  // date and time from jQueryUI//
+
+  var strDate = $.datepicker.formatDate('dd.mm.yy', new Date());
+  var strYear = $.datepicker.formatDate('yy', new Date());
+  $('#date').append(strDate);
+  $('#year').append(strYear);
+
+  // Svatky //
+
+  $.ajax({
+    url: "https://svatky.adresa.info/json",
+    type: 'GET',
+    dataType: 'json',
+    success: function (res) {
+      var parsed = JSON.parse(JSON.stringify(res));

```

Comment:
try/catch,
proc stringify?

Reply:
Stringify byl na základě příkladu z internetu, ale jednalo se o začátek kódu, ke kterému jsem se už nevrátil, v dalších ajaxech jsem to tak neřešil, i tady opraveno.

try catch - stačí použít success, error?

Reply:
try catch je kvuli json.parse